deploy/helm/templates/logs-daemon-set-windows.yaml

```diff
@@ -0,0 +1,143 @@
+                operator: NotIn
+                values:
+                - fargate
+      {{- end }}
+      nodeSelector:
+         kubernetes.io/os: windows
+      {{- if .Values.otel.logs.nodeSelector }}
+        {{ toYaml .Values.otel.logs.nodeSelector | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: swi-opentelemetry-collector
+          image: "{{ .Values.otel.image.repository }}:{{ .Values.otel.image.tag | default .Chart.AppVersion }}-nanoserver-ltsc2022"
+          imagePullPolicy: {{ .Values.otel.image.pullPolicy }}
+          command:
+            - c:\swi-otelcol.exe
+            - --config=c:\conf\relay.yaml
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: SOLARWINDS_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: solarwinds-api-token
+                  key: SOLARWINDS_API_TOKEN
+                  optional: true
+          envFrom:
+            - configMapRef:
+                name: {{ include "common.fullname" . }}-common-env
+{{- if .Values.otel.logs.telemetry.metrics.enabled }}
+          ports:
+            - name: http
+              containerPort: {{ (split ":" .Values.otel.logs.telemetry.metrics.address)._1 }}
+              protocol: TCP
+{{- end}}
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 13133
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 13133
+          resources:
+{{ toYaml .Values.otel.logs.resources | indent 12 }}
+          volumeMounts:
+            - mountPath: c:\var\log\pods
+              name: varlogpods
+              readOnly: true
+            - mountPath: c:\var\log\containers
+              name: varlogcontainers
+              readOnly: true
+            - mountPath: c:\var\lib\docker\containers
+              name: varlibdockercontainers
+              readOnly: true
+            - mountPath: c:\conf
+              name: opentelemetry-collector-configmap
+              readOnly: true
+            - name: logcheckpoints
+              mountPath: c:{{ .Values.otel.logs.filestorage.directory }}
+      volumes:
+        - name: varlogpods
+          hostPath:
+            path: c:\var\log\pods
+        - name: varlogcontainers
+          hostPath:
+            path: c:\var\log\containers
+        - name: varlibdockercontainers
+          hostPath:
+            path: c:\var\lib\docker\containers
+        - name: logcheckpoints
+          hostPath:
+            path: c:{{ printf "%s/%s" .Values.otel.logs.filestorage.directory .Values.cluster.uid }}
```

Comment:
We may need to sanitize the `.Values.cluster.uid` here. Windows file system is more strict about allowed path names and the `.Values.cluster.uid` can be set by customers to basically anything.

Reply:
good point, UID is GUID by default but it can be customized by customer. Hash will be better